Marlin/src/pins/stm32f1/pins_LONGER3D_LK.h

```diff
@@ -95,7 +95,11 @@
   #define FAN_MAX_PWM                        255
 #else
   #define FAST_PWM_FAN                            // STM32 Variant allow TIMER2 Hardware PWM
-  #define FAST_PWM_FAN_FREQUENCY           31400  // This frequency allow a good range, fan starts at 3%, half noise at 50%
+  #if FAST_PWM_FAN_FREQUENCY && FAST_PWM_FAN_FREQUENCY != 31400
+    #warning "FAST_PWM_FAN_FREQUENCY of 31400 is preferred!"
```

Comment:
Hmm, you can avoid/skip that for the Longer LKs, users may have to tweak it with custom fans... 31.4 is ok for factory one, but this part fan may die or may be replaced by another brand or model...

Reply:
Ok. It is important to remember a board can be placed into any printer, so the pins file should not make any assumptions about the machine it is installed in. That is the job of the example configurations.